localized_metadata/metadata_IT.js

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ * Copyright (C) 2010 The Libphonenumber Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+/**
+ * @fileoverview Generated metadata for file
+ * ../resources/PhoneNumberMetadata.xml
+ * @author Nikolaos Trogkanis
+ */
+
+goog.provide('i18n.phonenumbers.metadata');
+
+/**
```

Comment:
This is the Italian stuff. Please note that this is NOT minified. These are original metadata ready for compilation.